knox/service.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<service role="SUPERSETUI" name="supersetui" version="1.8.3">
+    <routes>
+        <route path="/rvf/">
+          <rewrite apply="SUPERSETUI/rvf/inbound/root" to="request.url"/>
```

Comment:
we should change role name `SUPERSETUI` to some other name. @arpit-agarwal @ankursinghal2005 @jitendra-kumawat  any more suggestions on the same?

Reply:
Sure. It can be done on Knox directly/Ansible
Keep it `RVFUI` for now

Reply:
Alos folder name on cluster has to be changed to align

Reply:
yes, it's deployment related changes we can set whatever we feel good.